scripts/runStart.js

```diff
@@ -13,7 +13,7 @@ for (let j = 2; j < process.argv.length; j++) {
 
 // Insert the chains you want to start here. Available values are:
 // 'Avalanche', 'Moonbeam', 'Polygon', 'Fantom', 'Ethereum'
-const chains = [];
+const chains = ['Avalanche', 'Ethereum'];
```

Comment:
Can you revert this change? By default, if chains array is empty, this script will spin up 5 chains. We'd like to keep that behavior.